setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
+name = zppy-e3sm-chem-diags
+version = 1.0.0
+author = Hsiang-He Lee
```

I'd suggest revising to 
```suggestion
author = Hsiang-He Lee, Qi Tang, Michael J. Prather, Jean-Christophe Golaz
```
for their contributions of the science and metrics design, original code and examples, and refactor the link with zppy.